HospitalWebApp/Resources/verification-mail.html

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+  <style type="text/css">
+    /* SPECIAL CLIENT STYLES */
+
+    #outlook a {
+      padding: 0;
+    }
+
+    /* Force Outlook to provide a "view in browser" message */
+
+    .ReadMsgBody {
+      width: 100%;
+    }
+
+    .ExternalClass {
+      width: 100%;
+    }
+
+    /* Force HM to display emails at full width */
+
+    .ExternalClass,
+    .ExternalClass p,
+    .ExternalClass span,
+    .ExternalClass font,
+    .ExternalClass td,
+    .ExternalClass div {
+      line-height: 100%;
+    }
+
+    /* Force HM to display normal line spacing */
+
+    body,
+    table,
+    td,
+    p,
+    a,
+    li,
+    blockquote {
+      -webkit-text-size-adjust: 100%;
+      -ms-text-size-adjust: 100%;
+    }
+
+    /* Prevent WebKit and Windows mobile changing default text sizes */
+
+    table,
+    td {
+      mso-table-lspace: 0pt;
+      mso-table-rspace: 0pt;
+    }
+
+    /* Remove spacing between tables in Outlook 2007 and up */
+
+    img {
+      -ms-interpolation-mode: bicubic;
+    }
+
+    /* Allow smoother rendering of resized image in Internet Explorer */
+
+    /* RESET STYLES */
+
+    body {
+      margin: 0;
+      padding: 0;
+    }
+
+    img {
+      border: 0 none;
+      height: auto;
+      line-height: 100%;
+      outline: none;
+      text-decoration: none;
+    }
+
+    a img {
+      border: 0 none;
+    }
+
+    .imageFix {
+      display: block;
+    }
+
+    table,
+    td {
+      border-collapse: collapse;
+    }
+
+    #bodyTable {
+      height: 100% !important;
+      margin: 0;
+      padding: 0;
+      width: 100% !important;
+    }
+
+    #footer a {
+      color: #00a4bd;
+      text-decoration: none;
+    }
+
+    /* Responsive Styles */
+
+    @media only screen and (max-width: 480px) {
+      .responsiveRow {
+        width: 100% !important;
+      }
+      .responsiveColumn {
+        display: block !important;
+        width: 100% !important;
+      }
+    }
+  </style>
+  <!--[if mso]><style type="text/css">body, table, td {
+  font-family: Helvetica Neue, Avenir Next, Arial, Helvetica, sans-serif !important;
+}</style><![endif]-->
```

Comment:
Will you use this style?

Reply:
Probably no, this comment came with this verification mail template.